tardis/plasma/base.py

```diff
@@ -258,10 +260,35 @@ def remove_hidden_properties(self, print_graph):
                 print_graph.remove_node(str(item.name))
         return print_graph
 
-class StandardPlasma(BasePlasma):
+    def to_hdf(self, path_or_buf, path='', close_hdf=True, filter=None):
```

Comment:
Well my only insight to this was [TEP002#Implementation](https://github.com/tardis-sn/tep/blob/master/TEP002_model_hdf5.rst#implementation), and specifically this part:

> This will also allow the higher level .to_hdf-methods to select specific information to store.
This might be grouped in hardcoded sets like "minimal" or "detailed". 

So, my thought was to have something similar to:
`minimal = FilterCollection(plasma=PlasmaFilter(...), runner=RunnerFilter(...), atom=AtomDataFilter(...))`

If you say that the only information that needs filtering are plasma properties, then, fine I'll remove the filter module. I can totally not provide you with a use case for extra filtering. At this point I don't even know which properties should be stored from all the objects:

>    Radial1Dmodel
    Plasma
    MontecarloRunner
    Simulation
    atomic data

I would even propose updating TEP002 with these information.